src/patternfly/components/Form/examples/form-select-input-example.hbs

```diff
@@ -1,35 +1,41 @@
 {{#> form}}
   {{#> form-group}}
     {{#> form-label for="select1"}}Select input{{/form-label}}
-    {{#> form-control}}
-      {{#> form-select id="select1" name="example-select1"}}
-        {{#> form-select-option value="Option 1"}}The first option{{/form-select-option}}
-        {{#> form-select-option value="Option 2"}}The second option{{/form-select-option}}
-      {{/form-select}}
-    {{/form-control}}
+    {{#> form-controls controlType="select" id="select1" name="selectexample1"}}
```

Comment:
In all but a few of your form fields (and excluding `type="radio"`), the `name` matches the `id`. I know it doesn't matter as far as the HTML is concerned, but for the example code, do you intend for the `name` to match the `id` (except for radio groups)? Just thought I'd mention it in case you intended for them to match.